donate/templates/pages/core/contributor_support_page.html

```diff
@@ -0,0 +1,96 @@
+{% extends "pages/base_page.html" %}
+{% load form_tags static wagtailcore_tags wagtailimages_tags i18n %}
+
+{% block content %}
+
+<header class="page-header">
+    <div class="page-header__container">
+        <h1 class="page-header__heading">{{ page.title }}</h1>
+    </div>
+</header>
+
+<div class="layout">
+    <div class="layout__container">
+        <div class="layout__full-col">
+            <div class="page-body support-form">
+            {% if 'submitted' in request.GET %}
+                <h1 class="heading heading--primary">
+                    {% trans "Thank you for contacting us." %}
+                </h1>
+
+                <br>
+
+                <div>
+                    <p class="rich-text">{% trans "We will get back to you soon." %}</p>
+                </div>
+            {% else %}
+                <form class="" action="https://mozillasalescloud--mkl.cs4.my.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8" method="POST">
+                    <p class="rich-text">{% blocktrans %}
+                        If you need help with a contribution, fill out this form and we'll get back to you soon.
+                    {% endblocktrans %}</p>
+
+                    <input type=hidden name="orgid" value="{{ orgid }}">
+                    <input type=hidden name="retURL" value="{{ page.full_url }}?submitted=true">
+
```

Comment:
```suggestion
<input type=hidden name="recordType" id="recordType" value="012P0000000SW6LIAW">
```

Reply:
is this is a fixed value or does it need a variable set for it?

Reply:
It's a fixed value.